File: fixtures/gold/stripe-app/heredoc-notes.php
<?php
// Positive control (Loop 300): a heredoc must not black out REAL code below
// it — an unterminated-looking body or the closer line must hand back code
// position so the genuine `use` + constructor lines keep being found.
$docBlurb = <<<TXT
This worker charges cards. A glob like /*.php appears here on purpose.
TXT;

use Stripe\StripeClient;

$client = new \Stripe\StripeClient(getenv('STRIPE_KEY'));
$client->charges->retrieve($chargeId);

File: fixtures/gold/stripe-app/heredoc_notes.rb
# Positive control (Loop 301): heredoc body quotes the OLD bootstrap
# (must not be sites); real code after the closer must be found.
NOTE = <<~DOC
  old bootstrap:
    require 'stripe'
  the DOC lookalike below must not terminate early
DOC
require 'stripe'

Stripe.api_key = ENV['STRIPE_KEY']

queue = []
queue << 'ITEM'
shifted = 1 << 2
